Stoccoin-Official/Stoccoin-Website
# feat: footer icons added and aligned with text

What this PR does?

fixes the missing Koo icon and aligns the social media links name text in the footer component.

Screenshot

![Screenshot 2023-06-14 000222](https://github.com/Stoccoin-Official/Stoccoin-Website/assets/125852599/9152e394-5821-4ae4-9a53-3a95b7f5c725)

Test

1. Go to landing page
2. Scroll down to footer section
3. Now you can see the changes
